### ISSUE TITLE
[4.6.3] PHP Fatal error in FullyQualifiedGlobalConstantsSniff

### ISSUE THREAD

**Issue by UncertaintyP** [slevomat/coding-standard#405]
Hi, I wanted to try out this coding standards but have problems getting it to work.

### Steps
1. Installed `slevomat/coding-standard` via composer locally as dev dependency
2. add `<rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard/ruleset.xml" />` to `ruleset.xml`
3. execute `phpcs app/User.php --standard=ruleset.xml`

### Error
**PHP Fatal error:  Class 'SlevomatCodingStandard\Sniffs\Namespaces\AbstractFullyQualifiedGlobalReference' not found in /home/heisenberg/code/klubbox/api/vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/Namespaces/FullyQualifiedGlobalConstantsSniff.php on line 8**
(the class exists in composers autoload_classmap)

### Annotations
Running phpcs with PSRX or Squiz rules does not throw any fatal error and CodeSniffer executes as expected.

### Versions
```
slevomat/coding-standard version 4.6.3
```

```
php --version
PHP 7.2.7-0ubuntu0.18.04.2 (cli) (built: Jul  4 2018 16:55:24) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.2.7-0ubuntu0.18.04.2, Copyright (c) 1999-2018, by Zend Technologies
```

```
phpcs --version
PHP_CodeSniffer version 3.3.1 (stable) by Squiz (http://www.squiz.net)
```

**Comment by kukulich** [comment]
Where is your ruleset.xml located?

**Comment by UncertaintyP** [comment]
Main directory of the project.

**Comment by kukulich** [comment]
I'm sorry, I'm not able to reproduce it.

**Comment by oldy777** [comment]
You should install phpcs locally in you project. It helps me

**Comment by UncertaintyP** [comment]
Thank you very much @oldy777 it is working with `vendor/bin/phpcs` (I checked both versions btw - they are the same). Strange thing is it is even working in Sublime3 although the path to phpcs is the global one.
